db/migrate/20200922230337_create_services.rb

```diff
@@ -0,0 +1,31 @@
+class CreateServices < ActiveRecord::Migration[4.2]
+  def self.up
+    create_table :services, id: :bigint do |t|
+      t.string :name, :null => false, :limit => 255, :required => true
+      t.string :ref, :null => false, :limit => 255, :required => true, :default => "master"
+    end
+    add_index :services, [:name], :unique => true, :name => 'on_name'
+
+    add_column :pushes, :service_id, :integer, :limit => 8, :null => false
+
+    add_index :pushes, [:service_id], :name => 'on_service_id'
+
+    web_service = Service.create!(name: 'web', ref: 'production')
+    Service.create!(
+      [
+        { name: 'rs_west',    ref: '2b7a8338fde0a998d8aa6b540f1aa4dcb3f9018f' },
+        { name: 'rs_east',    ref: '2b7a8338fde0a998d8aa6b540f1aa4dcb3f9018f' },
+        { name: 'rs_central', ref: '2b7a8338fde0a998d8aa6b540f1aa4dcb3f9018f' }
+      ]
+    )
```

Comment:
TIL I didn't know that `create!` would create multiple records if given an array, I wonder if this is shorthand for `3.times { Service.create! }` or if it's wrapping all of these in a transaction (so that if 1 failed, they would all be rolled back)

Reply:
According to [this post](https://til.hashrocket.com/posts/nnlah14d5s-array-of-hashes-create-many-activerecord-objects) it doesn't batch the insert statements, so I'm _guessing_ it isn't wrapping them in a transaction. Either way -> neat

Reply:
Also one thing to note with [these comments](https://github.com/Invoca/pre_deploy_checker/pull/20#discussion_r494017625), we'll still want to keep the seeds for the entries in `db/seeds.rb` because we'll want to have some data for `test` and `development` environments (plus if/when we create a pre-deploy checker in staging).

I'd suggest in that seeds.rb file to have a comment or a log statement that lets the user know that if they're going to use this for actual comparisons that they should likely update the sha's

Reply:
added them back in with a comment 👍